constraint-solver/src/symbolic_expression.rs

```diff
@@ -395,4 +395,26 @@ impl<T: FieldElement, V: Clone> SymbolicExpression<T, V> {
             )
         }
     }
+
+    /// Returns the multiplicative inverse in the field.
+    pub fn field_inverse(&self) -> Self {
+        if let SymbolicExpression::Concrete(x) = self {
+            assert!(x != &T::from(0));
+            SymbolicExpression::Concrete(T::from(1) / *x)
+        } else if let SymbolicExpression::BinaryOperation(x, BinaryOperator::Div, y, _) = self {
```

Comment:
Perhaps unrelated to this PR, but looking at `field_div`, I think it is possible to have an expression like `x / 0` (because the case where the LHS is unknown but the RHS is known to be 0 is not handled). In that case, inverse would turn it into `0 / x`, which should we `0` instead. But I guess the problem is that we can have `x / 0` in the first place.